keps/sig-apps/4443-configurable-pod-failure-policy-reasons/kep.yaml

```diff
@@ -0,0 +1,41 @@
+title: KEP Template
+kep-number: 4443
+authors:
+  - "@danielvegamyhre"
+owning-sig: sig-apps
+status: provisional
+creation-date: 2024-01-26
+reviewers:
+  - "@ahg-g"
+  - "@kannon92"
+approvers:
+  - "@alculquicondor"
+  - "@msau42"
```

Comment:
we can probably defer API review to implementation time

Reply:
Makes sense, updated